scripts/lib.sh

```diff
@@ -99,7 +99,7 @@ sudo-askpass() {
 }
 
 upgrade-pip() {
-    pip install --upgrade "pip==21.1.2" "wheel==0.36.2"
+    pip install --upgrade 'pip>=21.2'
```

Comment:
Was this getting on the way?

How come we're getting rid of `wheel`? In new venvs, there will be some packages that will need to be built from source.

Reply:
added it back